README.md

```diff
@@ -38,6 +38,7 @@ Use the binaries from [the release page](https://github.com/snyk-tech-services/j
     -dryRun=<true|false>                                                // Optional. result can be found in a json file were the tool is run
     -debug=<true|false>                                                 // Optional. enable debug mode
     -ifUpgradeAvailableOnly=<true|false>                                // Optional. create ticket only for upgradable issues
+    -configFile=<true|false>                                            // Optional. Use jira.yaml to set the config
```

Comment:
should we take the path to file here instead? Seems odd to have this as a boolean and then read a file from dick. Usually you would provide the path to file `--configFile=jira.yaml`

Reply:
agreed